src/buffer_manager.ts

```diff
@@ -615,7 +615,7 @@ export class BufferManager implements Disposable {
             vim.api.nvim_buf_set_name(bufId, bufname)
             vim.api.nvim_buf_set_option(bufId, "modifiable", not isExternalDoc)
             -- force nofile, just in case if the buffer was created externally
-            vim.api.nvim_buf_set_option(bufId, "buftype", "nofile")
+            vim.api.nvim_buf_set_option(bufId, "buftype", "acwrite")
```

Comment:
Perhaps add some comments here, as `acwrite` is not commonly used, and this PR adds code that depends on this option.

Reply:
Additionally, the original comments are outdated.